mmdet/core/hook/wandblogger_hook.py

```diff
@@ -109,6 +110,23 @@ def __init__(self,
         self.ckpt_hook: CheckpointHook = None
         self.eval_hook: EvalHook = None
 
+    def import_wandb(self):
+        try:
+            import wandb
+            from wandb import init  # noqa
+
+            # Fix ResourceWarning when calling wandb.log in wandb v0.12.10.
+            # https://github.com/wandb/client/issues/2837
+            if digit_version(wandb.__version__) < digit_version('0.12.10'):
+                raise RuntimeError('The wandb version is required to be '
```

Comment:
If we simply want to avoid warning, we should not raise runtimeerror but just raise a warning here.

Reply:
OK